packages/devtools_app/lib/src/memory/flutter/memory_snapshot_models.dart

```diff
@@ -110,6 +132,8 @@ class Reference extends TreeNode<Reference> {
 
   final bool isLibrary;
 
+  final bool isExternals;
```

Comment:
maybe rename isExternals to something more verbose or document why isExternals and isExternal are different.

Reply:
```
  /// The External heap (contains all external heap items).
  final bool isExternals;

  /// An External heap item.
  final bool isExternal;
```